### PR TITLE
Updates to admin-manual.md

### PR DESCRIPTION
**Pull request by callunity**

### CODE REVIEW

File: admin-manual.md [hunk 1]
```diff
@@ -26,18 +26,13 @@ Development Machine(s):
 - Eclipse
 - A merge tool such as Meld
 
-# Creating a Fork and Setting Configuration
+# Creating a Fork
 Use the GitHub interface or a private instance to create a forked repository.
 
-Proceed to set up a build environment, then edit
-`miso-web/src/main/webapp/WEB-INF/miso-config.xml` and set the naming schemes.
-The naming schemes will determine how MISO checks if object names (especially
```
Comment (apmasell) [inline admin-manual.md]: Writing naming schemes still matters. I'm not sure where it should go.

**Comment by apmasell** [comment]
:+1:

File: admin-manual.md (hunk 1 above)
Comment (callunity) [inline admin-manual.md]: Naming scheme beans in `miso-web/src/main/webapp/WEB-INF/miso-config.xml` are interpolations of the `miso.X.naming.scheme` values set in the miso.properties file. That's why I've moved this part to line 122 below.

File: admin-manual.md [hunk 2]
```diff
@@ -109,21 +104,32 @@ and populate it with the following information:
       <Parameter name="miso.propertiesFile" value="file:${CATALINA_HOME}/conf/Catalina/localhost/miso.properties" override="false"/>
     </Context>
 
+Make sure the database path in the `server.xml` is correct for your install:
```
Comment (djcooke) [inline admin-manual.md]: ROOT.xml?

**Comment by djcooke** [comment]
:+1: after server/ROOT.xml fix

**Pull request merged**